sale_product_pack/models/sale_order_line.py

```diff
@@ -71,13 +71,14 @@ def expand_pack_line(self, write=False):
                 else:
                     vals_list.append(vals)
             if vals_list:
-                self.create(vals_list)
+                return super().create(vals_list)
 
     @api.model_create_multi
     def create(self, vals_list):
         new_vals = []
         res = self.browse()
-        for elem in vals_list:
+        for idx, elem in enumerate(vals_list):
+            elem["sequence"] = idx
```

Comment:
Why doing it here? Prepare the sequence on `expand_pack_line`, summing the main line sequence + an incremental number

Reply:
This is still not correct, as you can make the create of the sales order lines on several batches, so you must check the resequence mechanism implemented in this version, and hook it to respect the relative pack lines orders.